runtime/resourcedictionary_test.go

```diff
@@ -252,11 +252,12 @@ func TestRuntimeResourceDictionaryValues(t *testing.T) {
      transaction {
 
          prepare(signer: AuthAccount) {
-             let c = signer.borrow<&Test.C>(from: /storage/c)!
```

Comment:
Here and below: Why do we have to load+save now and can't keep borrowing? Maybe keep the borrow and adjust the test cases to allow the operations on the reference, like it used to. 

For example, `forceInsert` was already introduced earlier in the same manner.

Reply:
ah this is no longer needed. Reverted in https://github.com/onflow/cadence/pull/2573/commits/b72a3b1f04e100888d52f2a59c8a6b47cc95b312